.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: krew-index CI
+
+on:
+- push
+- pull_request
+
+jobs:
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    env: 
+      KREW_VERSION: v0.3.3
+      GO111MODULE: on
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Install kubectl
+      run: |
+        sudo curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.1/bin/linux/amd64/kubectl
```

Comment:
you need `set -e`.
we discussed this before, only last command of a shell causes a shell failure. if curl fails here, this command will still succeed because you didn't do `set -e` or use &&.

Reply:
I already added it `set -euo pipefail`